Create a list of companies/organizations contributing to Kubeflow

jlewi opened the pull request:
Start a file to keep track of organizations participating in Kubeflow.

* With Kubeflow around the corner we want to be sure to aknowledge the
  companies/organizations who are participating in Kubeflow.

* I think starting a file in source control is the best way to manage
  this over time.

/cc @ddysher @mattf @ewilderj @aronchick @elsonrodriguez @inc0 @willingc

<!-- Reviewable:start -->
---
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/kubeflow/community/63)
<!-- Reviewable:end -->

ddysher commented:
Do we have plan to add it to a README or is there any tools to convert it to a website?

willingc commented on member_organizations.yaml:
```diff
@@ -0,0 +1,27 @@
+# This file provides information about companies/organizations
+# that would like to be aknowledged as contributing/participating in
```
s/aknowledged/acknowledged

inc0 commented on member_organizations.yaml:
```diff
@@ -0,0 +1,27 @@
+#
+# We do not infer organizations based on affiliation in members.yaml because
+# some members may be contributing as individuals and not part of a deeper involvement
+# with Kubeflow for their company/organization.
```
I'd add note that this document is entirely voluntary and companies might contribute as they like without adding footprint there. A lot of companies just contributes one or two patches total and, while being very valid contributions, are reluctant to make commitments

jlewi replied:
Isn't that covered by the first line of the file? "This file provides information about companies that would like to be acknowledged"
So if companies don't want to be acknowledged then they shouldn't add themselves.

jlewi commented:
Fixed; PTAL.

I also added instructions to our contributing.md.

@ddysher Yes we should auto-generate a page to be included in our website acknowledging contributors but that doesn't exist yet. The immediate goal is just being able to get an accurate list before Kubecon so that we can manually create slides that don't leave anyone out.

jlewi commented:
@ewilderj @aronchick @inc0 can someone please "/lgtm"

ewilderj commented on CONTRIBUTING.md:
```diff
@@ -66,8 +61,34 @@ the following tags:
+
+If you would like your company or organization to be acknowledged for contributing to
+Kubeflow or participatng in the community (being a user counts) please send a PR
+adding the relevant info to[member_organizations.yaml](https://github.com/kubeflow/community/blob/master/member_organizations.yaml).
```
add space between `to` and `[member_organizations.yaml]`

ewilderj commented:
/lgtm
